<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QWidget" name="ResetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResetPassword</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="ResetWindow">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="otpGenerate">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>122</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QFrame" name="coverFrameReset1">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="MidFrameReset1">
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>15</number>
             </property>
             <property name="rightMargin">
              <number>15</number>
             </property>
             <property name="bottomMargin">
              <number>15</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>Rockwell</family>
                </font>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; text-decoration: underline;&quot;&gt;Reset Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="ResetFrameReset1">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <family>Rockwell</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Enter Admin ID</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="UserID">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>Enter your Admin ID</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="sendOtp">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Sent OTP</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <family>Rockwell</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Enter OTP Received</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="otpBox">
                  <property name="placeholderText">
                   <string>OTP</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <spacer name="horizontalSpacer_10">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="verifyOtp">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Verify</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_9">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="4" column="0" colspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <spacer name="horizontalSpacer_11">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="ReturnToLogin">
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Return To Login</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_12">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>178</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resetPage">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="1">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>55</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>221</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QFrame" name="coverFrameReset2">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <widget class="QFrame" name="MidFrameReset2">
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>15</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item row="1" column="0">
              <widget class="QFrame" name="ResetFrameReset2">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_8">
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item row="2" column="0">
                 <widget class="QPushButton" name="ResetPasswordButton">
                  <property name="text">
                   <string>Reset Password</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="newPass">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="mouseTracking">
                   <bool>true</bool>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>Enter Password</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="confNewPass">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="placeholderText">
                   <string>Confirm Password</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>Rockwell</family>
                 <pointsize>20</pointsize>
                 <bold>true</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="text">
                <string>Reset Password</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>221</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>55</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
